README.md

```diff
@@ -59,11 +59,14 @@ For more details, see [Build](https://github.com/OmniSharp/omnisharp-roslyn/blob
 
 Add the following setting to your [User Settings or Workspace Settings](https://code.visualstudio.com/Docs/customization/userandworkspace).
 
+To consume the latest build of omnisharp from "master": 
+
 ``` JSON
 {
-  "omnisharp.path": "<Path to the omnisharp executable>"
+  "omnisharp.path": "latest"
```

Comment:
The only concern I have here is that it doesn't help with debugging or development of OmniSharp, which I think is the goal of these instructions. In other words, these instructions are to help folks who have built a version of OmniSharp on their box so they can try it out and debug it. Could you expand this?

Reply:
I meant, could you expand the description to include information about `latest` rather than replacing the existing instructions with `latest`? Since the instructions here are meant for developers who are actively working on OmniSharp, it feels like `latest` might not be what they want. Instead, they'll want to debug the version of OmniSharp that they just built on their dev machine. Does that make sense?

Reply:
Yes it does. I'll make the changes. Thanks!